src/Form/Form.php

```diff
@@ -70,7 +76,8 @@ class Form
     public function schema(Schema $schema = null)
     {
         if ($schema === null && empty($this->_schema)) {
-            $schema = $this->_buildSchema(new Schema());
+            $schema = new $this->_schemaClass;
```

Comment:
Nitpick: Can directly do pass the new instance to `_buildSchema()` instead of assigning to a variable.